src/azul/strings.py

```diff
@@ -48,20 +55,53 @@ def pluralize(word: str, count: int) -> str:
     return result
 
 
-def unwrap(string: str):
+def format_description(string: str) -> str:
```

Comment:
With the change to implementation and name, this is not a general string function anymore. It should go where `unwrap_description` used to be.

Reply:
Ideally, I would want to invoke this function as `api.description(…)`. This implies either an `import azul.openapi as api` in the referring module or that this function is defined in `src/azul/openapi/api.py` in conjunction with `from azul.openapi import api`. I am not sure. I think it is a bad smell if modules import from the `__init__.py` of the package they're in which most certainly will be necessary if this function is defined in `src/azul/openapi/__init__.py`. Let's do this, you move the function back, leave the name and I'll add a commit that gives the function a logical and concise name.